src/v3/src/transformer/selectAuthenticator/transformSelectAuthenticatorUnlockVerify.ts

```diff
@@ -78,8 +67,36 @@ export const transformSelectAuthenticatorUnlockVerify: IdxStepTransformer = ({
     },
   };
 
+  const authenticatorList: AuthenticatorButtonListElement = {
+    type: 'AuthenticatorButtonList',
+    options: { buttons: authenticatorButtons, dataSe: 'authenticator-verify-list' },
+  };
+
   const identifier = getUIElementWithName('identifier', uischema.elements) as FieldElement;
 
+  // TODO: OKTA-741832 Remove this fork once AUTH_POLICY_FOR_ACCOUNT_MANAGEMENT monolith FF goes GA
```

Comment:
It shouldn't have any time urgency behind it and can be made after GA. It's more of a tech debt item since once the FF goes GA we'll just end up with a dead code path for the faked flow

Reply:
Ahh got it, TODO for tech debt not for GA; sounds good